tests/analyzer/metadata/test_repository_integrity_mismatch.py

```diff
@@ -14,6 +14,19 @@ def test_no_github_links():
     assert message == "Could not find any GitHub url in the project's description"
 
 
+def test_no_good_homepage_link():
+    current_info = deepcopy(PYPI_PACKAGE_INFO)
+    current_info["info"]["home_page"] = ""
+    current_info["info"]["project_urls"] = {
+        "Download": "UNKNOWN",
+        "Homepage": "https://github.com/pypa/samplproject",
+        },
+    current_info["info"]["summary"] = "https://github.com/pypa/sampleproject"
+    detector = PypiIntegrityMismatchDetector()
+    match, message = detector.detect(current_info, name="mypackage", path="")
+    assert not match
+    assert message == "Could not find a good GitHub url in the project's description"
```

Comment:
nitpick: sounds a bit fragile (i.e. test will break when we change the message)

Reply:
perhaps use a constant or remove (optional)

Reply:
@christophetd I can remove! I was mirroring the messages from the other tests. Does it make sense to remove the asserts for the messages for all the tests that have it?

Reply:
ah, if it's like that in other tests then let's leave as is even if suboptional